fictadvisor-api/src/v2/utils/documentation/pageText/updateByKey.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiDocumentationParams } from '../decorators';
+import { PageTextsResponse } from '@fictadvisor/utils';
+import { DefaultForbiddenResponse, DefaultUnauthorizedResponse } from '../defaultResponses';
+
+export const PageTextDocumentationUpdateByKey: ApiDocumentationParams = {
```

Comment:
Name of the documentation object (PageTextDocumentationUpdateByKey) must be the same as corresponding controller method (PageTextDocumentationUpdateMany)